src-extra/edu/stanford/nlp/mt/service/tools/CoreNLPToJSON.java

```diff
@@ -51,7 +51,7 @@
   static {
     properties.put("annotators", "tokenize,ssplit,pos,lemma,ner,parse");
     properties.put("ssplit.eolonly", "true");
-    properties.put("tokenize.options", "invertible=true,ptb3Escaping=false,asciiQuotes=true,splitAssimilations=false");
```

Comment:
I had to Google`ASCII Quotes` 😅, are we removing this because we want to use Unicode quotes?

Reply:
The option was removed from the corenlp library, it's just not supported any more.  I'm not sure what it did.

Reply:
Hmm, actually maybe I should change this to `quotes=ascii`, which may still be supported per https://stanfordnlp.github.io/CoreNLP/tokenize.html

Reply:
Yeah, I'm switching this, I thought the option had been removed but they've just changed how it worked.